questionnaire/questionnaire/utils/taskRunner/tasks/sequential/index.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+async function runTasksSequentially(taskDefinitions, run) {
```

Comment:
Small "issue", but could we rename this concept to "series". I assume that the potential intention here is to extend this functionality to also have a concept of parallel execution.

`runTasksInSeries`, `runTasksInParallel`

Reply:
Any reason to prefer series over sequential? I felt sequential behaviour was pretty definitive.

I was going to have a complementary "concurrent" task. The semantics are slightly different to parallel.